util/index.ts

```diff
@@ -0,0 +1,188 @@
+import { ArgumentParser } from 'argparse';
+import inquirer, { Answers } from 'inquirer';
+import inquirerFuzzyPath, { FuzzyPathQuestionOptions } from 'inquirer-fuzzy-path';
+
+import { Lang, languages } from '../resources/languages';
+
+import { run as findMissingTranslations } from './find_missing_translations';
+import { default as generateEffectIds } from './gen_effect_id';
+import { default as translateTimeline } from './translate_timeline';
+
+declare module 'inquirer' {
+  interface QuestionMap<T> {
+    fuzzypath: FuzzyPathQuestionOptions<T>;
+  }
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const getArgument = <T = string>(obj: any, propertyName: string): T | undefined => {
+  if (typeof obj === 'undefined' || obj === null)
+    return;
+
+  if (!(propertyName in obj))
+    return;
+
+  const prop = (obj as { [s: string]: unknown })[propertyName] as T;
+
+  if (typeof prop === 'undefined' || prop === null)
+    return;
+
+  return prop;
+};
```

Comment:
```suggestion
const getArgument = <T>(obj: unknown, propertyName: string): T | undefined => {
  if (typeof obj !== 'object' || obj === null)
    return;
  if (!(propertyName in obj))
    return;

  const data = obj as { [key: string]: T };
  // TODO: it'd be nice to check typeof data here too?
  return data[propertyName];
};
```

Also, does this get used for anything but strings? If no, maybe changing `T` to be `string` and checking `typeof data`?